db/dataGenerator.js

```diff
@@ -0,0 +1,64 @@
+/* eslint-disable no-console */
+const faker = require('faker'); // GENERATE 100 RECORDS!
+const fs = require('file-system');
+const s3 = require('./s3.js');
+
+// TOP DATA [recipeName, thumbs, description, imageURL, photoBy]
+
+let top = 1;
+const topData = [];
+
+while (top <= 100) {
+  topData.push(`
+  {
+    "recipeID": ${top},
+    "name": "${faker.random.words()}",
+    "thumbs": ${Math.ceil(Math.random() * (100 - 54) + 54)},
+    "description": "${faker.lorem.paragraph()}",
+    "photo": {
+      "photoID": ${top},
+      "img": "${s3.images[top - 1]}",
+      "photoBy": "Photo By ${faker.name.findName().replace('\'', '')}"
+    }
+  }`);
+
+  top += 1;
+}
+
+fs.writeFile('./topData/topData.js',
+  `module.exports = myTopData = [${topData}
+]`,
+  (err) => {
+    if (err) return console.log(err);
+    return console.log('Wrote Top Data');
+  });
+
+
+// REVIEWS DATA [stars, text, name, location, date]
+
+let reviews = 1;
+const reviewData = [];
+
+
+while (reviews <= 10000) {
+  reviewData.push(`
+{
+  "reviewID": ${reviews},
+  "stars": ${Math.ceil(Math.random() * (5 - 0) + 0)},
```

Comment:
```suggestion
  "stars": ${faker.random.number(5)},
```
if your min is 0, you can pass a max value in for faker.random.number

Reply:
the min is actually 1 though because I'm using Math.ceil, if this would give me a zero at any point I think I need to keep it as is

Reply:
```suggestion
  "stars": ${Math.ceil(Math.random() * (5 - 0) + 0)},
```
"stars": ${faker.random.number({ min: 1, max:5)},

Reply:
```suggestion
  "stars": ${faker.random.number({ min: 1, max: 5 )},
```
this should work then, if you want to be consistent.